cypress/integration/parallel-2/productSwitch.spec.ts

```diff
@@ -221,11 +225,31 @@ describe('product switching', () => {
 		featureSwitches.accountOverviewNewLayout &&
 		featureSwitches.productSwitching
 	) {
-		it('navigates to product switching page from Account Overview', () => {
+		it('successfully completes product switching page from Account Overview', () => {
```

Comment:
Can you add a test to demonstrate a single contribution and multiple contributions, selecting something other than the first.